src/gen9_hevc_encoder.c

```diff
@@ -6570,7 +6570,11 @@ gen9_hevc_set_reflist(VADriverContextP ctx,
 
     for (i = 0; i < 16; i++) {
         frame_idx = -1;
-        obj_surface = SURFACE(ref_list[i].picture_id);
+        if (i < 15)
```

Comment:
it seems missed the i==15 case

Reply:
No, for the last i = 15, it goes to "else" and don't get the ref_list[i].

Reply:
it is ok to me